### PR TITLE
CEDS-3057 Message Inbox page

### PR DESCRIPTION
**Pull request by udinobi**

### CODE REVIEW

File: app/connectors/SecureMessagingConnector.scala [hunk 1]
```diff
@@ -26,7 +26,7 @@ import uk.gov.hmrc.http.HttpReads.Implicits.readRaw
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class SecureMessageFrontendConnector @Inject()(httpClient: HttpClient, config: SecureMessagingConfig)(implicit ec: ExecutionContext)
+class SecureMessagingConnector @Inject()(httpClient: HttpClient, config: SecureMessagingConfig)(implicit ec: ExecutionContext)
```
Comment (glidester) [inline app/connectors/SecureMessagingConnector.scala:29]: Why remove the 'Frontend' part of the name? This connector integrates with the 'secure-message-frontend' service, not the 'secure-message' (backend) service.

File: conf/application.conf [hunk 2]
```diff
@@ -91,10 +91,10 @@ microservice {
...
       host = localhost
       port = 9055
-      fetch-inbox = /secure-message-frontend/cds-file-upload-service/messages
-      fetch-message = /secure-message-frontend/cds-file-upload-service/conversation
-      submit-reply = /secure-message-frontend/cds-file-upload-service/conversation
-      reply-result = /secure-message-frontend/cds-file-upload-service/conversation/CLIENT_ID/CONVERSATION_ID/result
+      fetch-inbox = /secure-message-frontend/customs-declare-exports/messages
+      fetch-message = /secure-message-frontend/customs-declare-exports/conversation
+      submit-reply = /secure-message-frontend/customs-declare-exports/conversation
+      reply-result = /secure-message-frontend/customs-declare-exports/conversation/CLIENT_ID/CONVERSATION_ID/result
```
Comment (glidester) [inline conf/application.conf:97]: Whoopse! :grin: :+1:

File: app/connectors/SecureMessagingConnector.scala (hunk 1 above)
Comment (udinobi) [reply app/connectors/SecureMessagingConnector.scala]: I consider `Frontend` only as an implementation detail and `SecureMessagingConnector` more in line with the other "Secure Messaging" classes/topic (Controller and Config). Anyway, not a big deal. I have renamed it to the old name.

**Review by glidester — APPROVED** [review]
LGTM

**Pull request merged**